### PR TITLE
A few grammatical improvements

### PR DESCRIPTION
**Pull request by zoffixznet**
Most of these are just fixed comma splices.

Please review.

### CODE REVIEW

File: lib/Mojolicious/Guides/Tutorial.pod [hunk 1]
```diff
@@ -422,8 +422,8 @@ stash all the time.
 
 =head2 Restrictive placeholders
 
-The easiest way to make placeholders more restrictive are alternatives, you
-just make a list of possible values.
+The easiest way to make placeholders more restrictive is to use
+alternatives: you just make a list of possible values.
```
Comment (jberger) [inline lib/Mojolicious/Guides/Tutorial.pod]: this would be a semi-colon I believe, as many of these suggestions could also be

Comment (zoffixznet) [inline lib/Mojolicious/Guides/Tutorial.pod]: I feel a colon is more appropriate, as the second clause is not only related, but expounds the first clause. And yes, semicolons can be used to fix some of the comma splices, but I think in technical documentation that would feel a bit clunky, compared to concise, simple sentences.

Comment (kraih) [inline lib/Mojolicious/Guides/Tutorial.pod]: English is not my native language, but the colon looks odd to me.

**Comment by zoffixznet** [comment]
I have a "Grammatically Correct 2", as well as Strunk and White's style guide laying around somewhere. If you'd like, I could locate those books tomorrow and provide more support for my grammatical suggestions. I intend to read all of Mojo docs this month, so there are more PRs like this to come. Let me know.

**Comment by jberger** [comment]
Strunk and White may be a bit much, but I haven't used full colons in that
way. That said this (from a quick google) seems to support it:
http://www2.le.ac.uk/offices/ld/resources/writing/grammar/grammar-guides/semicolon

**Comment by zoffixznet** [comment]
Very well :)

Let me know how I can amend this PR to make it acceptable for merging and I shall do so when I get home.

**Comment by jberger** [comment]
I guess what I'm saying is, it is technically acceptable as-is, though I
wouldn't have written it that way myself.

File: lib/Mojolicious/Guides/Tutorial.pod (hunk 1 above)
Comment (Grinnz) [inline lib/Mojolicious/Guides/Tutorial.pod]: I could see either way, but I would err on the side of semi-colon in this case.

**Comment by zoffixznet** [comment]
I've changed the colon to semi-colon, based on the feedback from the group. Let me know if any other changes are needed.

File: lib/Mojolicious/Guides/Tutorial.pod [hunk 2]
```diff
@@ -422,8 +422,8 @@ stash all the time.
 
 =head2 Restrictive placeholders
 
-The easiest way to make placeholders more restrictive are alternatives, you
-just make a list of possible values.
+The easiest way to make placeholders more restrictive is to use
+alternatives; you just make a list of possible values.
```
Comment (kraih) [inline lib/Mojolicious/Guides/Tutorial.pod]: Looks like you did not check the line length after making changes.

**Comment by kraih** [comment]
My english is not good enough to judge the actual changes, but i'm afraid i have to vote :-1: for formatting issues.

**Comment by zoffixznet** [comment]
As I've mentioned, I found more similiar corrections to make, as well as a few grammaticall errors in words.

I'll merge add these changes to others when I'm done with all the docs and submit a new PR.

@kraih is right, I did not run the code through perltidy. I'll need to learn how to do that :)

**Comment by jhthorsen** [comment]
@zoffixznet:
```
$ cpanm App::githook::perltidy
$ cd mojo
$ githook-perltidy install
# do changes and perltidy is run on commit
```

**Comment by kraih** [comment]
Perltidy does not work for documentation, you'll have to check every line manually.

**Pull request closed without merging**